.github/workflows/tanzu-tile.yml

```diff
@@ -0,0 +1,56 @@
+name: Tanzu Tile
+
+# The workflow triggered by any change in deployments/cloudfoundry/bosh/
+# or /deployments/cloudfoundry/tile
+
+on:
+  pull_request:
+    paths:
+      - 'deployments/cloudfoundry/bosh/**'
+      - 'deployments/cloudfoundry/tile/**'
+
+permissions:
+  contents: write
+
+defaults:
+  run:
+    working-directory: 'deployments/cloudfoundry/tile'
+
+jobs:
+
+  test:
+    name: Test Tanzu Tile creation
+    # Use 20.04.5 until https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16450 is resolved
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v3
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Install BOSH CLI
+        shell: bash
+        run: |
+          brew install cloudfoundry/tap/bosh-cli
+          bosh -v
+      - name: Install PCF CLI
+        shell: bash
+        run: |
+          pip install tile-generator
+      - name: Run release script
+        shell: bash
+        run: |
+          ./make-latest-tile
+          FILE=""
+          # The tile will be in the product directory, named
+          # splunk-otel-collector-VERSION.pivotal
+          for f in $(find product -maxdepth 1 -regex "product\/splunk-otel-collector\-.*pivotal")
+          do
+            FILE=$f
```

Comment:
should the file be added as an artifact so that's it's available outside the action lifetime?

Reply:
I think yes, depending on when it's being run. The artifact (the tile itself) should be produced for every Splunk Collector release, so it would be good to have some kind of option for this when running the release job. The artifact shouldn't be kept around otherwise though.

I think this functionality should be done in another change, as we'll have to modify the release job in gitlab, since the github release being successful is a pre-requisite for building the tile.

Reply:
Discussed more with Ryan, I'll add the tile created by the workflow as an artifact so it can be checked after job runs as well.